infra/docker-compose/.env.sample

```diff
@@ -0,0 +1,16 @@
+COMPOSE_PROJECT_NAME=feast
+
+FEAST_VERSION=0.3.0
```

Comment:
Should we perhaps default to `latest` and just ensure that we always push the latest tagged docker image?

Reply:
Will do.